src/core/components/stock-management-menu-app-item/stock-management-menu-app-item.scss

```diff
@@ -0,0 +1,14 @@
+.customTile {
+    background-color: #f0f0f0;
+    border-radius: 4px;
+    padding: 20px;
+    width: 50px;
+    height: 100px;
+    text-align: center;
```

Comment:
this too

Reply:
@jabahum It's because it was my first time working on a ticket for stock mgt so everything came along